test/node/bigint_tests.ts

```diff
@@ -0,0 +1,123 @@
+import * as BSON from '../register-bson';
```

Comment:
Can you rename the file to `bigint.test.ts`, it's the name format that gets picked up by VSCode, and the pattern we use in the driver